src/Microsoft.AspNet.Razor/GeneratorResults.cs

```diff
@@ -2,7 +2,8 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System.Collections.Generic;
-using Microsoft.AspNet.Razor.Generator.Compiler;
+using Microsoft.AspNet.Razor.Chunks;
+using Microsoft.AspNet.Razor.CodeGeneration;
```

Comment:
This class should be in the `Microsoft.AspNet.Razor.CodeGenerator` namespace. Wouldn't need the `using` then.